From 9196249442d3d9b360b77a5bd35ffb6c49935c78 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Wed, 26 Sep 2012 13:12:20 +0100
Subject: [PATCH 048/364] * util/grub-mkconfig_lib.in
 (is_path_readable_by_grub): Redirect errors from grub-probe to /dev/null, not
 stdout.

---
 ChangeLog                 | 5 +++++
 util/grub-mkconfig_lib.in | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index d81a9a6..c43f8a4 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+2012-09-26  Colin Watson  <cjwatson@ubuntu.com>
+
+	* util/grub-mkconfig_lib.in (is_path_readable_by_grub): Redirect
+	errors from grub-probe to /dev/null, not stdout.
+
 2012-09-26  Vladimir Serbinenko  <phcoder@gmail.com>
 
 	* grub-core/fs/affs.c (grub_affs_mount): Support AFFS bootblock in
diff --git a/util/grub-mkconfig_lib.in b/util/grub-mkconfig_lib.in
index 8f21eb2..3574839 100644
--- a/util/grub-mkconfig_lib.in
+++ b/util/grub-mkconfig_lib.in
@@ -65,7 +65,7 @@ is_path_readable_by_grub ()
 
   # ... or if we can't figure out the abstraction module, for example if
   # memberlist fails on an LVM volume group.
-  if abstractions="`"${grub_probe}" -t abstraction "$path"`" 2>&1 ; then 
+  if abstractions="`"${grub_probe}" -t abstraction "$path"`" 2> /dev/null ; then 
       :
   else
     return 1
-- 
1.8.1.4

